app/templates/navbar.html

```diff
@@ -12,20 +12,37 @@
         </div>
         <div id="navbar" class="navbar-collapse collapse">
             <ul class="nav navbar-nav">
-                <li class="openrecords_nav"><a href="{{ url_for('request.new') }}">Request a Record</a></li>
-                <li class="openrecords_nav"><a href="{{ url_for('request.view_all') }}">Search Requests</a></li>
-                <li class="openrecords_nav"><a href="{{ url_for('report.show_report') }}">Reports</a></li>
-                <li class="openrecords_nav"><a href="{{ url_for('main.faq') }}">FAQ</a></li>
-                <li class="openrecords_nav"><a href="{{ url_for('main.about') }}">About</a></li>
-                <li class="openrecords_nav"><a href="{{ url_for('main.technical_support') }}">Technical Support</a></li>
+                <li class="openrecords_nav">
+                    <a href="{{ url_for('request.new') }}">Request a Record</a>
+                </li>
+                <li class="openrecords_nav">
+                    <a href="{{ url_for('request.view_all') }}">Search Requests</a>
+                </li>
+                <li class="openrecords_nav">
+                    <a href="{{ url_for('report.show_report') }}">Reports</a>
+                </li>
+                <li class="openrecords_nav">
+                    <a href="{{ url_for('main.faq') }}">FAQ</a>
+                </li>
+                <li class="openrecords_nav">
+                    <a href="{{ url_for('main.about') }}">About</a>
+                </li>
+                <li class="openrecords_nav">
+                    <a href="{{ url_for('main.technical_support') }}">Technical Support</a>
+                </li>
                 {% if not current_user.is_anonymous %}
-                    <li class="openrecords_nav"><a href="{{ url_for('auth.manage') }}">Manage Account</a></li>
-                    {% if (current_user.has_agency_admin and current_user.has_agency_active) or current_user.is_super %}
-                        <li class="openrecords_nav"><a href="{{ url_for('admin.main') }}">Agency Administration</a></li>
-                    {% endif %}
-                {% endif %}
+                <li class="openrecords_nav">
+                    <a href="{{ url_for('auth.manage') }}">Manage Account</a>
+                </li>
+                {% if (current_user.has_agency_admin and current_user.has_agency_active) or current_user.is_super %}
```

Comment:
Can you indent this second `if` block inside the first `if` block and then align the `endif`s